membership-attribute-service/app/services/GuardianPatronService.scala

```diff
@@ -8,7 +8,7 @@ import com.gu.memsub.Product.GuardianPatron
 import com.gu.memsub.Subscription._
 import com.gu.memsub._
 import com.gu.memsub.subsv2.ReaderType.Direct
-import com.gu.memsub.subsv2.{CovariantNonEmptyList, PaidCharge, PaidSubscriptionPlan, Subscription}
+import com.gu.memsub.subsv2.{CovariantNonEmptyList, SingleCharge, Subscription, SubscriptionPlan}
```

Comment:
PaidSubscriptionPlan is called SubscriptionPlan
PaidCharge is called SingleCharge (to make it more easily searchable I didn't just go for "Charge")

feel free to make other suggestions though as it's easy enough to tweak right now

Reply:
SubscriptionCharge maybe?

Reply:
I will do it in another PR as it's going to cause a lot of conflicts

Reply:
https://github.com/guardian/members-data-api/pull/1076